app/presenters/pafs_core/validation_presenter.rb

```diff
@@ -19,7 +19,16 @@ def complete?
     end
 
     def project_name_complete?
-      # NOTE: currently has to be present before creation
+      return add_error(:project_name, "Tell us the project name") if name.blank?
+
+      unless name =~ PafsCore::ProjectNameStep::PROJECT_NAME_REGEX
+        return add_error(:project_name, "The project name must only contain letters, underscores, hyphens and numbers")
+      end
+
+      if PafsCore::Project.where.not(id: project.id).exists?(name: name)
```

Comment:
The `.not` threw me - please add a comment explaining that this is to find any *other* projects with the same name.